src/test/system_tests/test_bucket_access.js

```diff
@@ -107,6 +107,8 @@ function setup() {
 
     let account;
     return P.resolve()
+        // Create a default pool for the users.
+        .then(() => test_utils.create_hosts_pool(client, POOL_NAME))
```

Comment:
```suggestion
        .then(() => test_utils.create_hosts_pool(client, POOL_NAME, 3))
```

Reply:
@nimrod-becker Why not use the default?

Reply:
Easier to understand the test flow when it's explicit